minecraft.js

```diff
@@ -66,6 +66,25 @@ app.post('/command', (req, res) => {
   }, 250);
 });
 
+app.get('/properties', (req, res) => {
+  const text = fs.readFileSync('./server.properties').toString('utf-8');
+  const result = fileToJSON(text);
+  res.send(result);
+});
+
+app.put('/properties', (req, res) => {
+  const properties = req.body;
+  let string = '#Minecraft server properties\n#(last boot timestamp)\n';
+  for (const property in properties) {
```

Comment:
When using a `for...in` type loop typically I prefer to do `for (const key in properties)`

Reply:
Makes it more clear you're referencing the key rather than the actual property.

Reply:
Alrighty, that change has been made.